proto/telemetry.proto

```diff
@@ -19,6 +19,16 @@ enum TelemetryNodeType {
   TELEMETRY_NODE_TYPE_COMPACTOR = 4;
 }
 
+enum TelemetryClusterType {
+  TELEMETRY_CLUSTER_TYPE_UNSPECIFIED = 0;
+  TELEMETRY_CLUSTER_TYPE_SINGLE_NODE = 1;
```

Comment:
No need to have both `SINGLE_NODE` and `STANDALONE`.

- `risingwave single-node` or `risingwave` is the actual way to run a single-process RisingWave instance from operating system directly without container.
- `risingwave standalone` is only used in container (docker compose or Kubernetes), to run multiple roles in one container. In another word, the `TelemetryClusterType` should never be `STANDALONE`, but should be either `KUBERNETES` or `DOCKER_COMPOSE` instead.